Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -32,19 +32,16 @@
 #include "AuthenticationManager.h"
 #include "AuxiliaryProcessMessages.h"
 #include "DataReference.h"
-#include "Download.h"
```

Comment:
Why all the changes to this cross platform file? Did you accidentally use an older version?